HKMP/Game/Client/PlayerManager.cs

```diff
@@ -218,49 +416,35 @@ internal class PlayerManager {
             Team team,
             byte skinId
         ) {
-            if (playerData.PlayerContainer != null) {
-                Logger.Get().Warn(this, $"We already have created a player container for ID {playerData.Id}, removing");
-                Object.Destroy(playerData.PlayerContainer);
-            }
-
-            // Create a player container
-            var playerContainer = new GameObject($"Player Container {playerData.Id}");
-            playerContainer.transform.position = new Vector3(position.X, position.Y);
+            GameObject playerContainer;
 
-            playerContainer.AddComponent<PositionInterpolation>();
+            foreach (var data in _playerData) {
```

Comment:
This is an empty loop, I'm assuming it can be remove or you may have forgot to write something here.